.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,15 @@
+---
+name: 💡 Feature Request
```

Comment:
TBD: Need to agree on (potential) emoji and the name

(this is what is used in [Swift Crypto](https://github.com/apple/swift-crypto/blob/master/.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md))